### PR TITLE
Added default stage to tree view

### PR DESCRIPTION
**Pull request by PCatinean**
Boolean field was missing from tree view making it harder to identify default stages.

### CODE REVIEW

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Please change module version according https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

**Comment by PCatinean** [comment]
Done

File: project_task_default_stage/__manifest__.py
Comment (pedrobaeza) [inline project_task_default_stage/__manifest__.py]: It should be 10.0.1.1.0

Comment (PCatinean) [reply project_task_default_stage/__manifest__.py]: Thought the same since it's a small change though it matched better the middle version that said it should require module update. Also if someone should do a position after the same element it would change the design slightly.

But if you think it should increment the last number I can make the change, your call.

Comment (pedrobaeza) [reply project_task_default_stage/__manifest__.py]: Yeah, but I think you have read incorrectly the guide. The change I propose is for changes that require a module update.

Comment (PCatinean) [reply project_task_default_stage/__manifest__.py]: Ah, I read it corectly but the 0 from 10.0 got me confused.

**Review by pedrobaeza — APPROVED** [review]
Thanks

**Review by rafaelbn — APPROVED** [review]
Tested in runbot 👍

**Review by dreispt — APPROVED** [review]

**Pull request merged**
